docs/data/data-grid/row-updates/row-updates.md

```diff
@@ -39,6 +39,14 @@ Multiple row updates at a time are supported in [Pro](/x/introduction/licensing/
 
 ## Infinite loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
+:::warning
+This feature is deprecated and will be removed in `v9.x`.
```

Comment:
Should this change accompanied with `unstable_dataSource` => `dataSource` to be relevant with https://github.com/mui/mui-x/pull/13878#discussion_r1831298844?
We could even make it part of https://github.com/mui/mui-x/issues/15488 to do both together.
Wdyt?

Reply:
Are we going to mark the data source as stable?
If yes, then this can stay like this and I can remove the deprecation warnings from the cherry-pick PR
If no, then I need to revert this and it should be part of the linked issue

Reply:
> Are we going to mark the data source as stable?

For me, yes! Probably double-checking with @joserodolfofreitas would be a nice idea.
In any case, my point was to do both the changes (deprecating the previous API + marking the data source stable) together to not having to revert one of them later. So to avoid the rush on the decision we could revert the change now and make it an action item for #15488

> from the cherry-pick PR

Do we want to release this to v7 too?
The approach we followed up until v5 and v6 was to only port critical bug fixes (and some handpicked features) to the current major, the rest goes to next, basically to increase the reasons for adoption.

Reply:
> my point was to do both the changes (deprecating the previous API + marking the data source stable) together to not having to revert one of them later, in case we decide against it

agree. I will revert it and action points to https://github.com/mui/mui-x/issues/15488

> Do we want to release this to v7 too?
I intended it to, since it was planned and started long time ago.
@joserodolfofreitas what is your take on this?

Reply:
updated to `unstable_lazyLoading`

Reply:
I was thinking about it, but then again, it is just a number and I don't see how it can be changed in the future so I left it as stable

Reply:
I have updated it, since unstable allows us to potentially remove the API, and `lazyLoadingRequestThrottleMs` is related to `lazyLoading`, so they have to be unstable/stable together

Reply:
Cool, thanks!

Reply:
Cool. Thanks for the update.